src/ProjectTemplates/Web.ProjectTemplates/content/Components-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,293 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Web",
+    "Blazor",
+    "WebAssembly",
+    "PWA"
+  ],
+  "name": "Blazor Web App",
+  "defaultName": "BlazorApp",
+  "description": "A project template for creating a Blazor web app that supports both server-side rendering and client interactivity. This template can be used for web apps with rich dynamic user interfaces (UIs).",
+  "groupIdentity": "Microsoft.Web.Blazor",
+  "precedence": "9800",
+  "guids": [
+    "4C26868E-5E7C-458D-82E3-040509D0C71F",
+    "5990939C-7E7B-4CFA-86FF-44CA5756498A",
+    "650B3CE7-2E93-4CC4-9F46-466686815EAA",
+    "0AFFA7FD-4E37-4636-AB91-3753E746DB98"
+  ],
+  "identity": "Microsoft.Web.Blazor.CSharp.8.0",
+  "thirdPartyNotices": "https://aka.ms/aspnetcore/8.0-third-party-notices",
+  "preferNameDirectory": true,
+  "primaryOutputs": [
+    {
+      "path": "Components-CSharp.csproj"
+    }
+  ],
+  "shortName": "blazor",
```

Comment:
```suggestion
  "shortName": "blazorweb",
```

Reply:
I would rather keep it blazor. There is only going to be `blazor` and `standaloneblazor`. I don't think the suffix adds anything

Reply:
Let say at some point we come up with a way to have a single Blazor project target web, mobile, and desktop from a single project. Would we want to use the `blazor` short name for that template or this web specific one?

Reply:
Let's stick with blazor for now and discuss that offline. It's an easy change that we can make at any point.